Makefile

```diff
@@ -167,6 +168,10 @@ lint-front-eslint: ## lint TypeScript sources
 	@$(YARN) lint
 .PHONY: lint-front-eslint
 
+run-storybook: ## Run front-end's storybook
+	@$(COMPOSE) up storybook
+.PHONY: run-storybook
+
```

Comment:
To be homegeneous, I'll up this service in detach mode then add a command `stop-storybook` to stop the container like we are doing with app (`make run`, `make stop`)